sdc11073/sdcclient/subscription.py

```diff
@@ -626,7 +626,7 @@ def stop(self, closeAllConnections=True):
         self.httpd.socket.close()
         if closeAllConnections:
             if self.httpd.dispatcher is not None:
-                self.httpd.dispatcher.methods = None
+                self.httpd.dispatcher.methods = {}
```

Comment:
What about `HttpServerThread`?

Reply:
No, what I meant was: Why is this changed here, but not in the `HttpServerThread`, which has the exact same code in `stop()`?

Reply:
ah, I see. It turned out that I could delete that line in HttpServerThread completely, because that dispatcher has no methods member.